lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -345,7 +345,10 @@ def do_install hosts, opts = {}
               if host['roles'].include?('frictionless') &&  (! version_is_less(version, '3.2.0'))
                 # If We're *not* running the classic installer, we want
                 # to make sure the master has packages for us.
-                deploy_frictionless_to_master(host)
+                # Only for pre 3.99
+                if version_is_less(version, '3.99')
+                  deploy_frictionless_to_master(host)
```

Comment:
Using the puppet-dashboard rake tasks to classify a pe_repo platform only works prior to 3.99, but you still need to perform this classification on 4.0.  You just can't use the rake tasks for it.